magma/bits.py

```diff
@@ -542,7 +566,21 @@ def simulate(self, value_store, state_store):
 BitsType = Bits
 
 
-class UInt(Bits):
+class Number(Bits):
```

Comment:
pref `Int` over `Number`.

On that note, some of the other common circuit generator functions can prob be moved here, like `add` if we dispatch on `cls` (that way it will be still be specialized for `SInt` vs `UInt` but the logic is generic). Not critical to do that now.